src/components/workflowEditor/Workflow.tsx

```diff
@@ -97,6 +103,19 @@ export class Workflow extends Component<WorkflowProps, WorkflowState> {
         }
     }
 
+    goToWorkFlowEditor = (node: NodeAttr) => {
+        if (node.branch === GIT_BRANCH_NOT_CONFIGURED) {
```

Comment:
same here

Reply:
done, except this.props.id was coming as number so needed to convert to string since the function expects a string value